lib/http/index.js

```diff
@@ -1,22 +1,18 @@
 const express = require('express');
-const Coinsence = require('../coinsence');
-const CoinsenceKit = require('../kit');
 const ethers = require('ethers');
 const bodyParser = require('body-parser');
-const fs = require('fs');
-const path = require('path');
 const morgan = require('morgan');
+const Sentry = require('@sentry/node');
+
+Sentry.init({ dsn: 'https://95d2d147a3124da7a38b70360c1bdf95@sentry.io/1488845' });
```

Comment:
we should have this configurable via an ENV variable. and not have deploy/account specific data in the rep. 
also please note this is public.

Reply:
yeah sorry missed that